src/Profile/ProfileScreen.js

```diff
@@ -170,104 +197,125 @@ export default function ProfileScreen({ navigation }) {
           Alert.alert(err.error, err.message);
         }
       });
+      const updatedUser = {
+        ...currentUser, email, address, phoneNumber: phoneNum,
+      };
+      serviceUpdateUser(updatedUser);
+    } else if (!validateEmail(email)) {
+      Alert.alert('Invalid email submitted.');
+    } else if (phoneNum.length !== 14) {
+      Alert.alert('Invalid phone number submitted.');
     } else {
-      Alert.alert('Please fill out all fields.');
+      Alert.alert('Please fill out the address field.');
```

Comment:
I personally think it should be kept this way because a user doesn't need to see an alert if there are no changes (if someone just changes the icon, it's a bit weird to have this message). There is also an alert that notifies the user that changes have been saved, so I feel as though this might not be necessary